docs/source/scripts/plot_contour.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+import optuna
+import plotly
```

Comment:
Please separate the third party library and the local library (i.e., optuna). (c.f., https://github.com/optuna/optuna/issues/514)